resources/puppetlabs/lein-ezbake/template/foss/ext/redhat/init.erb

```diff
@@ -74,21 +71,12 @@ start() {
     <%= action %>
     <% end -%>
 
-    pushd "${INSTALL_DIR}" &> /dev/null
```

Comment:
I think if we try to start in a directory we don't own, bad things sometimes happen. Probably worth putting this back in.